docs/user-guide/global-configurations/filter-condition.md

```diff
@@ -0,0 +1,132 @@
+# Filter Condition
+
+[![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+## Introduction
+
+The [workflows](../creating-application/workflow/) you create in Devtron for managing the CI-CD of your application can be made flexible, for e.g., not all events (such as image builds) generated during the CI stage require progression to the CD stage. Therefore, instead of creating multiple pipelines that cater to complex requirements, Devtron provides you the option of creating filters to make your workflow efficient. 
+
+Using filter conditions, you can control the progression of events. Here are a few general examples:
+* Images containing the label "test" should not be eligible for deployment in production environment
+* Only images having tag versions greater than v0.7.4 should be eligible for deployment
+* Images hosted on Docker Hub should be eligible but not the rest
+
+---
+
+## Steps to Create a Filter
+
+{% hint style="info" %}
+**Pre-requisite**: You must have application(s) with CI-CD workflow(s) configured
+{% endhint %}
+
+1. From the left sidebar, go to **Global Configurations** → **Filter Condition**.
+
+2. Add a filter condition.
+
+    ![Figure 1: Creating Our First Filter](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/filters/add-filter.jpg)
+
+3. In the **Define Filter condition** section, you get the following fields:
+    * **Filter For**: Choose the pipeline upon which the filter should apply. Currently, you can use filter conditions for CD pipelines only. Support for CI pipelines is underway.
+
+    ![Figure 2: 'Define Filter Condition' section](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/filters/define-filter-page.jpg)
+
+    * **Filter Name**: Give a name to the filter.
+    * **Description**: (Optional) Add a description to the filter, preferably explaining what it does.
+    * **Filter Condition**: You can specify either a pass condition, fail condition, or both the conditions:
+        * **Pass Condition**: Only those events that satisfy the condition will be directed through your pipeline.
+        * **Fail Condition**: Events that fail the condition will not be directed through your pipeline..
+    * **Use CEL Expression**: You can use `Common Expression Language` (CEL) to define the conditions. Currently, you can create conditions around the following criteria:
+        * **containerImage**: Package that contains all the necessary files and instructions to run an application in a container, e.g., gcr.io/k8s-minikube/kicbase:v0.0.39
```

Comment:
The container image variable returns a string value, follows the format of `<registry>/<repository>:<tag>` E.g: gcr.io/k8s-minikube/kicbase:v0.0.39